### PR TITLE
Kucerar/3164 account detail

### PR DESCRIPTION
**Pull request by kucera-roman**
https://redmine.czechidm.com/issues/3164

### CODE REVIEW

File: Realization/frontend/czechidm-acc/src/locales/cs.json [hunk 1]
```diff
@@ -3473,6 +3525,62 @@
...
+        "one": {
+          "name": "Základní nastavení účtu",
+          "help": "Udělejte základní nastavení účtů a vyberte vlastníka",
+          "system": "Systém",
+          "type": "Typ uživatele",
+          "owner": "Vlastník"
+        },
+        "two": {
+          "name": "Atributy",
+          "help": "Vyplňte atributy účtu. Hodnoty mohou být předvyplněny navrhovanými hodnotami. Pokud je to dovoleno, hodnoty mohou být ručně upraveny.",
```
Comment (tomdoischer) [inline Realization/frontend/czechidm-acc/src/locales/cs.json:3569]: The if part is not accurate, it is always allowed right now I think.

File: Realization/frontend/czechidm-acc/src/locales/en.json [hunk 2]
```diff
@@ -3470,6 +3522,62 @@
...
+        "one": {
+          "name": "Basic setting for account",
+          "help": "Do basic configuration of account and select owner",
+          "system": "System",
+          "type": "Type of user",
+          "owner": "Owner"
+        },
+        "two": {
+          "name": "Attributes",
+          "help": "Fill attributes for account. Values could be filled with suggested values. If it's allowed, you can change them manually.",
```
Comment (tomdoischer) [inline Realization/frontend/czechidm-acc/src/locales/en.json:3538]: The if part is not accurate, it is always allowed right now I think.

**Review by tomdoischer — CHANGES_REQUESTED** [review]
Notes from testing

- on the provisioning detail for an account, active provisioning bulk actions do not work - doesn't always happen, see the console.log in the attached file
[log.txt](https://github.com/bcvsolutions/CzechIdMng/files/9820782/log.txt)
- provisioning detail for other accounts does not work correctly - requests are shown for different accounts
-  issue with overriding values with multiple provisioning mappings: DefaultSysSystemMappingService:425 - using the first provisioning mapping
- account detail - make the field for lines longer or dynamic - see the screenshot for a very long value - most of the page is empty and I have to scroll
- mapping and attribute override - if an attribute is removed from the mapping without account resave from the identity, the attribute is still seen on the account detail and will never disappear

File: Realization/backend/acc/src/main/java/eu/bcvsolutions/idm/acc/event/processor/AccountProvisioningProcessor.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import eu.bcvsolutions.idm.core.eav.api.dto.IdmFormInstanceDto;
+
+/**
+ * EAV account save. After it's saved we will do provisioning
+ * 
+ * @author Roman Kucera
+ */
+@Component("accAccountProvisioningProcessor")
+@Description("Do provisioning after eav is saved")
+public class AccountProvisioningProcessor extends CoreEventProcessor<IdmFormInstanceDto> {
```
[the remark this answers is not in the document]
Comment (kucera-roman) [reply Realization/backend/acc/src/main/java/eu/bcvsolutions/idm/acc/event/processor/AccountProvisioningProcessor.java]: fixed

**Comment by kucera-roman** [comment]
Fixing review notes
> Notes from testing
> 
> * on the provisioning detail for an account, active provisioning bulk actions do not work - doesn't always happen, see the console.log in the attached file
>   [log.txt](https://github.com/bcvsolutions/CzechIdMng/files/9820782/log.txt)
> * provisioning detail for other accounts does not work correctly - requests are shown for different accounts
> * ~~issue with overriding values with multiple provisioning mappings: DefaultSysSystemMappingService:425 - using the first provisioning mapping~~
> * account detail - make the field for lines longer or dynamic - see the screenshot for a very long value - most of the page is empty and I have to scroll
> * mapping and attribute override - if an attribute is removed from the mapping without account resave from the identity, the attribute is still seen on the account detail and will never disappear
> * ~~cannot create a new account via wizard because the AccountProvisioningProcessor catches it tries to do a provisioning without system entity~~

File: Realization/frontend/czechidm-acc/module-descriptor.js [hunk 4]
```diff
@@ -400,6 +400,79 @@ module.exports = {
...
+            titleKey: 'acc:content.accounts.detail.tabs.provisioning',
+            order: 15,
+            priority: 0,
+            icon: 'fa:circle-o',
+            path: '/account/:entityId/provisioning',
+            access: [ { type: 'HAS_ANY_AUTHORITY', authorities: ['ACCOUNT_READ'] } ],
+            items: []
+          },
+          {
+            id: 'account-other',
```
[the remark this answers is not in the document]
Comment (kucera-roman) [reply Realization/frontend/czechidm-acc/module-descriptor.js]: fixed, renamed id, renamed class

File: Realization/frontend/czechidm-acc/src/locales/cs.json (hunk 1 above)
Comment (kucera-roman) [reply Realization/frontend/czechidm-acc/src/locales/cs.json:3569]: After discussion this message is ok

File: Realization/frontend/czechidm-acc/src/locales/en.json (hunk 2 above)
Comment (kucera-roman) [reply Realization/frontend/czechidm-acc/src/locales/en.json:3538]: After discussion this message is ok

**Comment by tomdoischer** [comment]
Looks good to me!

**Pull request merged**